src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -4398,6 +4398,12 @@ internal static IEnumerable<CompletionResult> CompleteFilename(CompletionContext
                         if (CompletionRequiresQuotes(completionText, !useLiteralPath))
```

Comment:
@kwkam, your last commit had 2 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/ADOPTERS.md.https://docs.universaldashboard.io should work
```powershell
Name or service not known
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 114
114:                                 $null = Invoke-WebRequest -Uri $url -RetryIntervalSec 10 -MaximumRetryCount 6

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32 should work
```powershell
Failed to complete request to "https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```